elements/nuxeo-document-comments/nuxeo-document-comment-thread.html

```diff
@@ -286,7 +288,10 @@
         this._fetchComments(this.allCommentsLoaded);
       },
 
-      _submitReply: function() {
+      _submitReply: function(e) {
+        if(e) {
```

Comment:
missing space between if and condition:
```suggestion
        if (e) {
```